packages/devtools_app/lib/src/screens/debugger/controls.dart

```diff
@@ -33,9 +33,10 @@ class _DebuggingControlsState extends State<DebuggingControls>
   void didChangeDependencies() {
     super.didChangeDependencies();
     if (!initController()) return;
-    addAutoDisposeListener(
-      serviceManager.isolateManager.mainIsolateState!.isPaused,
-    );
+    final mainIsolateState = serviceManager.isolateManager.mainIsolateState;
```

Comment:
instead of doing a null check on all of these, just use `?` instead of `!`. addAutoDisposeListener will return early if the listenable is null.